### ISSUE TITLE
kompile -- old error message reported

### ISSUE THREAD

**Issue by grosu** [kframework/k-legacy#155]
There is no -l option anymore:

```
fslwork$ kompile logik2.k 
[Error] Compiler: Could not find main module 'LOGIK2'. Use -l(ang) option to
specify another.
    File:
/home/grosu/k/samples/java_rewrite_engine/tutorial/2_languages/4_logik/logik2.k
    Location: File system.
```

**Comment by osa1** [comment]
I think `-l(ang)` is replaced with `--main-module`, is that correct?

**Comment by radumereuta** [comment]
I believe that is correct.

**Comment by grosu** [comment]
But that was not the point of my issue ... The wrong error message was the problem.

**Comment by osa1** [comment]
What would be correct error message here? I thought the problem is wrong argument name in error message.

**Comment by grosu** [comment]
Yes, that was the problem and the only thing that needs to be fixed.

**Comment by osa1** [comment]
Fixed db9be80.
